src/api/quantum/enums/Side.java

```diff
@@ -0,0 +1,19 @@
+package api.quantum.enums;
+
+/**
+ * Represents a side of API
+ */
+public enum Side {
+    /**
+     * Represents clientside
+     */
+    CLIENT,
+    /**
+     * Represents serverside
+     */
+    SERVER,
+    /**
+     * Represents a object that is both compatible between clientside and serverside
+     */
+    UNIVERSAL,
```

Comment:
Then I have question: is there a point to universal that omitting it can't already say?